packages/create-xmcp-app/src/helpers/create.ts

```diff
@@ -18,6 +19,7 @@ interface ProjectOptions {
   useLocalXmcp?: boolean;
   deployToVercel?: boolean;
   skipInstall?: boolean;
+  initializeGit?: boolean;
```

Comment:
I think you may have code from your other PR here

Reply:
Yes, as Git would be first priority I had those changes. Therefore changed this to Draft for now
```
Note: Bun development branch has code from https://github.com/basementstudio/xmcp/pull/62 to avoid conflicts. If requested can recreate PR after merge.
```